omod-1.12/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_12/OrderGroupController1_12Test.java

```diff
@@ -0,0 +1,158 @@
+		Encounter encounter = orderGroup.getEncounter();
+		OrderSet orderSet = orderGroup.getOrderSet();
+		Order order = Context.getOrderService().getOrder(1);
+		Order order2 = Context.getOrderService().getOrder(3);
+		List<Order> orders = new ArrayList<Order>();
+		orders.add(order);
+		orders.add(order2);
+		
+		orderGroup.setOrders(orders);
+		order.setOrderGroup(orderGroup);
+		order2.setOrderGroup(orderGroup);
+		
+		MockHttpServletRequest req = request(RequestMethod.GET, getURI() + "/" + RestTestConstants1_12.ORDER_GROUP_UUID);
+		SimpleObject response = deserialize(handle(req));
+		Assert.assertEquals(patient.getUuid(), Util.getByPath(response, "patient/uuid"));
+		Assert.assertEquals(encounter.getUuid(), Util.getByPath(response, "encounter/uuid"));
+		Assert.assertEquals(orderSet.getUuid(), Util.getByPath(response, "orderSet/uuid"));
+		Assert.assertEquals(RestTestConstants1_12.ORDER_GROUP_DISPLAY, Util.getByPath(response, "display"));
+		List<Order> orderList = (List<Order>) PropertyUtils.getProperty(response, "orders");
+		Assert.assertEquals(2, orderList.size());
+	}
+	
+	@Test
+	public void shouldAddOrdersToAnExistingOrderGroup() throws Exception {
+		final String JSON_ORDER = " { \"orders\": [\n" + "    {\n"
+		        + "      \"encounter\": \"33d70956-b359-452a-b3da-b69c8ab459ce\",\n"
+		        + "      \"orderType\": \"2d3fb1d0-ae06-22e3-a5e2-0140211c9a66\",\n" + "      \"type\": \"order\",\n"
+		        + "      \"action\": \"NEW\",\n" + "      \"accessionNumber\": \"string\",\n"
+		        + "      \"dateActivated\": \"2012-09-11\",\n" + "      \"scheduledDate\": \"2012-09-11\",\n"
+		        + "      \"patient\": \"19887889-7110-46a7-a9f0-ebf1eb4bbd48\",\n"
+		        + "      \"concept\": \"985ff1a2-c2ef-49fd-836f-8a1d936d9ef9\",\n"
+		        + "      \"careSetting\": \"6f0c9a92-6f24-11e3-af88-005056821db0\",\n"
+		        + "      \"autoExpireDate\": \"2012-09-21\",\n"
+		        + "      \"orderer\": \"c2299800-cca9-11e0-9572-0800200c9a66\",\n" + "      \"previousOrder\": \"\",\n"
+		        + "      \"urgency\": \"ROUTINE\",\n" + "      \"orderReason\": \"\",\n"
+		        + "      \"orderReasonNonCoded\": \"for Test\",\n" + "      \"instructions\": \"string\",\n"
+		        + "      \"commentToFulfiller\": \"string\"\n" + "    }\n" + "  ]\n" + "}";
+		Integer ordersBefore = orderService.getOrderGroupByUuid(RestTestConstants1_12.ORDER_GROUP_UUID).getOrders().size();
+		Assert.assertEquals(0, orderService.getOrderGroupByUuid(RestTestConstants1_12.ORDER_GROUP_UUID).getOrders().size());
+		
+		MockHttpServletRequest req = newPostRequest(getURI() + "/" + RestTestConstants1_12.ORDER_GROUP_UUID, JSON_ORDER);
+		handle(req);
+		Integer ordersAfter = orderService.getOrderGroupByUuid(RestTestConstants1_12.ORDER_GROUP_UUID).getOrders().size();
+		Assert.assertEquals(1, orderService.getOrderGroupByUuid(RestTestConstants1_12.ORDER_GROUP_UUID).getOrders().size());
+		Assert.assertEquals(++ ordersBefore , ordersAfter);
```

Comment:
Is there any reason why you put space between the ++ and ordersBefore?

Reply:
i removed it